pkg/blob/controllerserver.go

```diff
@@ -112,6 +112,7 @@ func (d *Driver) CreateVolume(ctx context.Context, req *csi.CreateVolumeRequest)
 	}
 	if IsAzureStackCloud(d.cloud) {
 		accountKind = string(storage.Storage)
+		storageAccountType = "Standard_LRS"
```

Comment:
shall we report error here?

Reply:
Can we give users errors when they use the wrong account type? I've mentioned this in the README that under the hood the account type will be converted to `Standard_LRS` anyways. I thought for Azure Stack scenarios, simplicity rather than complexity is better.